docs/WARP-GRAPH.md

```
@@ -0,0 +1,1199 @@
+use warp_snapshot::wsc::types::{NodeRow, EdgeRow, Range, OutEdgeRef, AttRow};
+use warp_snapshot::wsc::write::OneWarpInput;
+
+pub fn build_one_warp_input(store: &crate::graph::GraphStore, root_node_id: NodeId) -> OneWarpInput<'static> {
+    // 1) NODES: already sorted by NodeId because BTreeMap
+    let nodes: Vec<(NodeId, NodeRecord)> = store
+        .nodes
+        .iter()
+        .map(|(id, rec)| (*id, rec.clone()))
+        .collect();
+
+    let node_rows: Vec<NodeRow> = nodes
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)